### ISSUE TITLE
💬 [Support Request] 

### ISSUE THREAD

**Issue by teckkev** [cmangos/issues#3554]
### Core Version

mangos-wotlk

### Support Request Details

Following instructions (https://github.com/cmangos/issues/wiki/Installation-Instructions) on Ubuntu 20.04.  Everything in the walk-through is smooth and reasonable to understand.  The issue and confusion comes at (Install databases) with Mysql.  I have installed Mysql, added user/pw, rights, and flushed.  Tested user on Mysql and Mysql Workbench to be valid @localhost.  

I ran the sudo ./InstallFullDB.sh and filled in the sheet. 
-----------------------------------------------------------------------------
Database host...........: localhost
Database port...........: 3306
MySQL user..............: mangos (password is defined in InstallFullDB.config)
MySQL user IP access....: localhost
MySQL binary path.......: /usr/bin/mysql
MySQL dump binary path..: /usr/bin/mysqldump
Core path...............: /home/wowemu/cmangos/mangos
World database name.....: wotlkmangos
Character database name.: wotlkcharacters
Realm database name.....: wotlkrealmd
Logs database name......: wotlklogs
LOCALES.................: YES
DEV_UPDATES.............: NO
AHBOT...................: NO
---------------------------------------------------------------------------

I hit option 1 to install ALL:
---------------------------------------------------------------------------
ll previous changes will be lost. Type 'DeleteAll' if you are sure : DeleteAll

> Creating wowemu user in database ... FAILED!
>>> ERROR 1045 (28000): Access denied for user 'mangos'@'localhost' (using password: YES)
Press space to continue...
----------------------------------------------------------------------------

I even edit line 54/55 to match my 'main' user/pw in InstallFullDB.sh

I have a SWGEMU server on Ubuntu using MariaDB with no issues.
 I am far from an expert, lol..  but I'm no newbie either.  Should I follow the manual creation steps in Walk-through or am I missing a step to your Automated Setup?

NOTE: I fresh installed UBUNTU and CMANGOS x2.  Both times during (git clone https://github.com/cmangos/mangos-wotlk.git mangos) I did get 2 script errors but finished at 100%. I know that could be 'normal'  for such project as Cmangos.  If not and you would like a new ticket made, I will be happy to do another install and record the 2 script errors.

### Steps to Reproduce

1.Fresh install Ubuntu 20.04
2.Follow step-by-step of https://github.com/cmangos/issues/wiki/Installation-Instructions
3.Equipment: 2023 Dell Optiplex7000 (VM using 8 Cores/16GB Memory)
4.

### Core SHA1 Commit Hash

na

### Database SHA1 Commit Hash

na

### Server Operating System

Ubuntu 20.04 LTS under Oracle VM

### Client Version

3.3.5a (Wrath of the Lich King)

### Client Operating System

Win11, Oracle VM, Ubuntu 20.04 LTS

**Comment by insunaa** [comment]
In `./InstallFullDB.sh` you first select `4`
then you specify the root user and root password
Continue following the instructions and it should set up the complete database for your automatically.

**Comment by teckkev** [comment]
> In `./InstallFullDB.sh` you first select `4` then you specify the root user and root password Continue following the instructions and it should set up the complete database for your automatically.

Thanks for response :)  Actually, I had to fix Mysql server for it wasn't saving anything unless I used Mysql DB to store it.  After I figured that out, DB installed just fine. 

FIXED: User Error setting up Mysql Server.  I followed the link below to fix my issue.

https://askubuntu.com/questions/1263000/mysql-access-denied-for-user-rootlocalhost-ubuntu-20-04-and-mysql8
